packages/pwa-buildpack/bin/buildpack

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env node
 
 require('yargs')
-    .commandDir('../dist/cli')
+    .commandDir('../lib/cli')
```

Comment:
Looks like this will resolve #1425 

@jcalcaben we won't need to add `yarn build` to the instructions.
cc @ericerway